docs/docs/genai/tracing/integrations/listing/fireworksai.mdx

```diff
@@ -0,0 +1,307 @@
+---
+sidebar_position: 12
+sidebar_label: FireworksAI
+---
+
+import { APILink } from "@site/src/components/APILink";
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import ImageBox from "@site/src/components/ImageBox";
+
+# Intro to FireworksAI
+
+FireworksAI is an inference and customization engine for open source AI. It provides day zero access to the latest SOTA OSS models and allows developers to build lightning fast magical AI applications. For more details on FireworksAI see [their docs](https://fireworks.ai/docs/getting-started/introduction).
+For a list of available models see [the model library](https://fireworks.ai/models).
+
+# Tracing FireworksAI
+
+[MLflow Tracing](/genai/tracing) provides automatic tracing capability for FireworksAI through the OpenAI SDK compatibility. FireworksAI is [OpenAI SDK compatible](https://fireworks.ai/docs/tools-sdks/openai-compatibility#openai-compatibility), you can use the <APILink fn="mlflow.openai.autolog" /> function to enable automatic tracing. MLflow will capture traces for LLM invocations and log them to the active MLflow Experiment.
+
+## Quick Start
+
+```python
+import openai
+import mlflow
+
+# Enable auto-tracing
+mlflow.openai.autolog()
+
+# Create an OpenAI client configured for FireworksAI
+openai_client = openai.OpenAI(
+    base_url="https://api.fireworks.ai/inference/v1",
+    api_key="your-fireworks-api-key"
```

Comment:
Might want to read sensitive keys from a vault or ENV VAR instead of plaintext in source code.